actors/builtin/market/market_actor.go

```diff
@@ -82,26 +82,24 @@ func (a Actor) WithdrawBalance(rt Runtime, params *WithdrawBalanceParams) *adt.E
 	amountExtracted := abi.NewTokenAmount(0)
 	var st State
 	rt.State().Transaction(&st, func() interface{} {
+		lt, err := adt.AsBalanceTable(adt.AsStore(rt), st.LockedTable)
```

Comment:
Ah is this to save a flush call on an unwritten locked funds table?  I'm not sure flushes are expensive if there is nothing to write so it might be worth the cleanliness to add to the mutation.

Reply:
This is done.